src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+// Actions
+const ADD = 'bookstore/books/add';
+const DEL = 'bookstore/books/del';
+
+// Empty array of books
+const initialState = [];
+
+// Reducer
+export default function booksReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case ADD:
+      return [...state, {
+        ...action.payload,
+      }];
+    case DEL:
+      return state.filter((book) => book.id !== action.id);
+    default: return state;
+  }
+}
+
+// Action Creators
+export function add(book) {
+  return {
+    type: ADD,
+    payload: { book },
+  };
+}
```

Comment:
- [ ] It would be better if you export your reducers as default. 

![Capture](https://user-images.githubusercontent.com/64173189/161600863-eb57e03c-4b37-4740-91fb-497fcf309995.JPG)

```suggestion
export default function add(book) {
  return {
    type: ADD,
    payload: { book },
  };
}
```

Reply:
Hello, thanks for the review ❤️ 
reducer is exported as default https://github.com/ZahraArshia/bookstore/blob/613542ad84d37907cbbbf191d8e308845c99eb59/src/redux/books/books.js#L9
those two functions are action creators